app/data/jsonSchemaForm/fundingAgreementSchema.ts

```diff
@@ -79,6 +102,20 @@ export const fundingAgreementUiSchema = {
     "ui:widget": "NumberWidget",
     isMoney: true,
   },
+  anticipatedFundingAmountPerFiscalYear: {
+    items: {
+      amount: {
+        "ui:widget": "text",
+        calculatedValueFormContextProperty: "calculatedAnticipatedAmount",
+      },
+    },
+    "ui:options": {
+      addable: false,
+      orderable: false,
+      removable: false,
+      label: false,
+    },
+  },
```

Comment:
I don't think this is needed if we're using the `ArrayFieldTemplate` but I left it in for now in case this is a better approach.